src/zodbupdate/tests.py

```diff
@@ -809,17 +835,17 @@ def test_encode_binary_leaves_none_untouched(self):
         mock['foo'] = None
         encoder = encode_binary('foo')
         result = encoder(mock)
-        self.assertEquals(result, False)
-        self.assertEquals(mock['foo'], None)
+        self.assertEqual(result, False)
+        self.assertEqual(mock['foo'], None)
```

Comment:
Minor: might as well use `assertFalse` and `assertIsNone`.

Reply:
Will do. I replaced `assertEquals` originally only because it was showing deprecation warnings.